libflux/flux-core/src/formatter/mod.rs

```diff
@@ -1399,21 +1399,10 @@ impl<'doc> Formatter<'doc> {
         loop {
             match (operators.next(), arguments.next()) {
                 (Some(operator), Some(arg)) => {
-                    doc = docs![
-                        arena,
-                        doc,
-                        docs![
-                            arena,
-                            operator,
-                            arg,
-                            if arguments.peek().is_some() {
-                                line.clone()
-                            } else {
-                                arena.nil()
-                            }
-                        ]
-                        .group(),
-                    ];
+                    doc += docs![arena, operator, arg].group();
+                    if arguments.peek().is_some() {
+                        doc += line.clone();
```

Comment:
This line needed to be outside of the group, otherwise the `|> call()` bit otherwise the group specified here would make the line think it fit.